docs/source/frameworks/onnx.rst

```diff
@@ -0,0 +1,460 @@
+====
+ONNX
+====
+
+
+Preface
+-------
+
+ONNX is an open format built to represent machine learning models. ONNX provides `high interoperability <https://onnx.ai/supported-tools.html#buildModel>`_  among various frameworks, as well as enable machine learning practitioners to maximize models' performance across `different hardware <https://onnx.ai/supported-tools.html#deployModel>`_.
+
+Due to its high interoperability among frameworks, we recommend you to check out the framework integration with ONNX as it will contain specific recommendation and requirements for that given framework.
+
+
+.. tab:: PyTorch
+
+   - `Quick tutorial about exporting a model from PyTorch to ONNX <https://pytorch.org/tutorials/advanced/super_resolution_with_onnxruntime.html>`_ from official PyTorch documentation.
+   - `torch.onnx <https://pytorch.org/docs/stable/onnx.html>`_ from official PyTorch documentation. Pay special attention to section **Avoiding Pitfalls**, **Limitations** and **Frequently Asked Questions**.
+
+.. tab:: TensorFlow
+
+   - `tensorflow-onnx (tf2onnx) <https://github.com/onnx/tensorflow-onnx>`_ documentation.
+
+.. tab:: Scikit Learn
```

Comment:
lets make all the tab in here https://sphinx-design.readthedocs.io/en/furo-theme/tabs.html#synchronised-tabs